EvolvSwiftSDK/EvolvSwiftSDKTests/DataModel-Tests/ConfigurationDataTest.swift

```diff
@@ -34,6 +34,6 @@ class ConfigurationDataTest: XCTestCase {
         let jsonData = json.data(using: .utf8)!
         let configurationData: Configuration = try! JSONDecoder().decode(Configuration.self, from: jsonData)
         
-        XCTAssertEqual(true, configurationData.experiments[0].home?.isEntryPoint)
+//        XCTAssertEqual(true, configurationData.experiments[0].home?.isEntryPoint)
```

Comment:
Why commenting this out?

Reply:
Sorry, forgot to uncomment after some changes.